lighthouse-plugin-publisher-ads/audits/first-ad-paint.js

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const AdPaintTime = require('../computed/ad-paint-time');
+const ComputedAdPaintTime = require('../computed/ad-paint-time');
```

Comment:
Can we rename "ad paint" to "ad render" in a future PR? It's not really accurate after this change

Reply:
Created #122 for this